### DISCUSSION TITLE
How can I use model.transcribe for multiple concurrent request

### DISCUSSION THREAD

**Discussion by Guneetsinghtuli** [openai/whisper#1390] [Q&A]
I am running a flask server with whisper 

```
def detect_audio(audio,filepath):
  try:

      japanese = model.transcribe(filepath)
      return japanese
  except Exception as e: 
      raise Exception('could not detect the audio :',e)
```

on running multiple concurrent request to the server I am getting an exception 

```The size of tensor a must match the size of tensor at non-singleton dimension```

It works fine for a single request but not with concurrent multiple request.

**Comment by mattcwebster** [comment]
Im look for an answer on this same topic
